tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/ServiceRecipientsSectionControllerTests.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NHSD.BuyingCatalogue.Ordering.Api.Controllers;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders;
+using NHSD.BuyingCatalogue.Ordering.Application.Persistence;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class ServiceRecipientsSectionControllerTests
+    {
+        [Test]
+        public void Ctor_NullRepository_Throws()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                var _ = new ServiceRecipientsSectionController(null, null);

```

Comment:
Not sure which constructor parameter this test is trying to verify. It might be clearer to have two tests one that supplies a null for the first parameter & the other test that supplies a null to the second. This way if a test fails it should be obvious which parameter didn't throw the correct exception.

Reply:
👍